file.c

```diff
@@ -7431,93 +7431,373 @@ Init_File(void)
     /*
      * Document-module: File::Constants
      *
-     * File::Constants provides file-related constants.  All possible
-     * file constants are listed in the documentation but they may not all
-     * be present on your platform.
+     * \Module \File::Constants defines file-related constants.
```

Comment:
How about mark as a code, if you don't like to make it a link?

```suggestion
     * \Module +File::Constants+ defines file-related constants.
```